### ISSUE TITLE
Sanity check focusable items under cookie notification 2.4.11 Focus Appearance (Minimum)

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#2015]
Sanity check focusable items under cookie notification 2.4.11 Focus Appearance (Minimum)

When a focusable item is under a (semi transparant) cookie notification (laying on top of, and at the bottom of a page), will this fail this SC?

In this case they are "completely hidden / behind the notification (like lot's of footer links)

**Comment by mraccess77** [comment]
If the control behind receives focus and it's totally obscured then it seems to fail the proposed wording of SC 2.4.11.     I personally likely also say it fails 2.4.7.

**Comment by jake-abma** [comment]
Thx @mraccess77 that is my impression too!

This will be of huge impact though I think as lots of sites have this, encountered it twice today when asked for a quick scan. Expect lots of pushback when really out there...

Do we need / want to also mention this specifically? Provide options for this? Examples?

They are often pretty much "in the face" and will not go away until a choice is made / even a required mandatory thing.

**Comment by jake-abma** [comment]
(and I did not even start the focus order thingy... :-) )

**Comment by jake-abma** [comment]
Luckily we have an example in the Deque Demo pages too: https://dequeuniversity.com/demo/dream if we want to discuss it / provide an example

**Comment by jake-abma** [comment]
check test page: https://new.ingwb.com/

Notification first in DOM, optional choice

- Focus indicators behind notification in footer
- Focus direct in notification but at bottom
- Zoom 200% / 400% / Landscape on mobile => blocking big parts of page

**Comment by jake-abma** [comment]
relates to: #2016

**Comment by alastc** [comment]
> check test page: https://new.ingwb.com/

I'd say that's a good case for adding scroll-padding to the bottom, which is removed when the cookie banner is removed.

Given that it is slightly transparent, it probably passes 2.4.7.

If it were not transparent, it would fail 2.4.7 & focus-appearance. 

I guess the question is whether we should apply the change-of-contrast check for 'under' the cookie banner?

My first thought is no, check the declared styles, if they pass then it passes those bullets. So it could pass focus-appearance min but not the AAA enhanced version.

**Comment by mraccess77** [comment]
It is one thing if a focus indicator is semi-transparent temporarily - but if the semi-transparency can not be changed to show the intended CSS values by shifting focus then my personal opinion is that the visually rendered version colors should be tested for SCs related to focus and non-text contrast.

**Comment by alastc** [comment]
@mraccess77 no disagreement, I was assuming that the usual case (once the cookie banner is closed), that you'd get the declared styles.

@jake-abma can we close this? I'm not seeing an update to make.
